pdr_backend/cli/cli_arguments.py

```diff
@@ -200,6 +240,58 @@ def __init__(self, description: str, command_name: str):
         )
 
 
+@enforce_types
+class _ArgParser_NUM_PPSS_NETWORK(
+    CustomArgParser,
+    NUM_Mixin,
+    PPSS_Mixin,
+    NETWORK_Mixin,
+):  # pylint: disable=too-many-ancestors
+    @enforce_types
+    def __init__(self, description: str, command_name: str):
+        super().__init__(description=description)
+        self.add_argument("command", choices=[command_name])
```

Comment:
please use add_arguments_bulk instead, it is **much** more concise. Applies to the other parsers as well

Reply:
also, please add tests for each of these in test_cli_arguments.py

Reply:
```also, please add tests for each of these in test_cli_arguments.py```

I see that `defined_parsers` and `test_arg_parser` would already cover these.